ban-importer/src/ban-fetcher.js

```diff
@@ -122,8 +122,9 @@ export default class BanFetcher {
             expires: expires,
             expired: !(ban.attributes.expires === null || expires.getTime() > Date.now()),
 
-            reason: classifyBanReason(`${ban.attributes.reason} ${ban.attributes.note}`),
-            rawReason: `${ban.attributes.reason} ${ban.attributes.note}`,
+            reason: classifyBanReason(ban.attributes.reason,ban.attributes.note),
+            rawReason: ban.attributes.reason,
+            rawNotes: ban.attributes.note,
```

Comment:
rawNotes here is not part of the DB schema

The object we are building here eventually bubbles down to what we store in the DB.

This leaves 3 options:

Update the Schema and DB.
Omit the Notes.
Store the rawReason as it was previously in the db (ban.reason + ban.notes)


When testing classifier changes, I rescan the raw reason fields as stored in the DB, with this change we will lose that ability unless we update the schema. 

https://github.com/CommunityBanList/Communitybanlist/blob/master/scbl-lib/db/models/ban.js#L10

Reply:
You are correct in that. I think we could get away with it without a DB schema change because we only reclassify unknown bans anyway, am I right?
In that case we could just put notes and reason into the rawReason field in the DB like it was before. Danger here being if we change that aspect further down the line, we might run into unforeseen consequences with this.

On the other hand, it would be more clean to separate them imo.
How hard is it to update the schema?